pkg/engine/avsets.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
+
+import (
+	"fmt"
+
+	"github.com/Azure/aks-engine/pkg/api"
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func CreateAvailabilitySet(cs *api.ContainerService, isManagedDisks bool) AvailabilitySetARM {
+
+	armResource := ARMResource{
+		APIVersion: "[variables('apiVersionCompute')]",
+	}
+
+	avSet := compute.AvailabilitySet{
```

Comment:
how about `as`?  since we use that in other places in the code and that's generally the Azure abbreviation for availability set

Reply:
normally I'd prefer brevity, but `as` doesn't really fit here.